charts/ocis/values.yaml

```diff
@@ -152,6 +152,8 @@ features:
   # -- Create demo users on the first startup.
   # Not recommended for production installations.
   demoUsers: false
+  # ID of the service account
+  serviceAccountID: "7641d020-719c-408d-be9b-dd0c2713fb9c"
```

Comment:
For the other UUIDs  (storage-uuid, graph application-id) we're using config maps.  See: https://doc.owncloud.com/ocis/next/deployment/container/orchestration/orchestration.html#mandatory-configmaps

IMO we should do the some for the serviceAccount UUID. Let's try to not ship default UUIDs in the values.yaml.

Reply:
Excellent idea! Implemented as suggested